Mutagen.Bethesda.Skyrim/Records/Common Subrecords/CanFlyHereConditionData_Generated.cs

```
@@ -454,6 +454,25 @@ public static implicit operator TranslationMask(bool defaultOn)
         }
         #endregion
 
+        #region Mutagen
+        public object? Parameter1
+        {
+            get => null;
```

Getting null here feels fine.  Feels a bit odd to just do nothing in the set.   Gut is telling me we should throw if the given value is not null (similar to how we throw if the type is wrong for used parameters)

---

I was worrying this could lead to strange behavior when you are working on generic conditions. Setting a property might not be intuitively something where you expect it throws when doing it? What do you think? You can of course just add a try catch block when you get any issues with this.

---

Yeah, not a great way either route... but I think that's just the price of writing generic API that is trying to handle everything.

The downside of the current setup is that users will expect that if they set the field and it "went through", that something was modified.  But instead, it didnt modify anything, and was just a "silent error" of sorts.

The throw at least is something the user can detect/handle, vs the silent error just leads to more of an undetectable paranoia, imo